libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -917,6 +917,8 @@ namespace SohImGui {
                     }
                     EnhancementCheckbox("N64 Mode", "gN64Mode");
                     Tooltip("Sets aspect ratio to 4:3 and lowers resolution to 240p, the N64's native resolution");
+                    EnhancementCheckbox("Disable Black Bar Letterboxes", "gDisableBlackBars");
+                    Tooltip("Disables Black Bar Letterboxes during cutscenes and Z-targeting. Note: there may be minor visual glitches that were covered up by the black bars. Please disable this setting before reporting a bug.");
```

Comment:
```suggestion
                    Tooltip("Disables Black Bar Letterboxes during cutscenes and Z-targeting\nNote: there may be minor visual glitches that were covered up by the black bars. Please disable this setting before reporting a bug");
```

Add a new line here, so the tooltip isn't too long, and remove the periods to keep it consistent with the rest of the tooltips.

Reply:
Good point. Thankyou